src/main/java/frc/trigon/robot/subsystems/swerve/swervemodule/SwerveModuleConstants.java

```diff
@@ -0,0 +1,135 @@
+package frc.trigon.robot.subsystems.swerve.swervemodule;
+
+import com.ctre.phoenix6.configs.CANcoderConfiguration;
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
+import com.ctre.phoenix6.signals.FeedbackSensorSourceValue;
+import com.ctre.phoenix6.signals.InvertedValue;
+import com.ctre.phoenix6.signals.NeutralModeValue;
+import com.ctre.phoenix6.signals.SensorDirectionValue;
+import edu.wpi.first.math.system.plant.DCMotor;
+import edu.wpi.first.units.Units;
+import edu.wpi.first.wpilibj2.command.sysid.SysIdRoutine;
+import org.trigon.hardware.RobotHardwareStats;
+import org.trigon.hardware.simulation.SimpleMotorSimulation;
+
+public class SwerveModuleConstants {
+    private static final double
+            DRIVE_MOTOR_GEAR_RATIO = 6.12,
+            STEER_MOTOR_GEAR_RATIO = 12.8;
+    private static final double
+            DRIVE_MOTOR_OPEN_LOOP_RAMP_RATE = RobotHardwareStats.isSimulation() ? 0.1 : 0.1,
+            DRIVE_MOTOR_CLOSED_LOOP_RAMP_RATE = RobotHardwareStats.isSimulation() ? 0.1 : 0.1;
+    private static final InvertedValue
+            DRIVE_MOTOR_INVERTED_VALUE = InvertedValue.CounterClockwise_Positive,
+            STEER_MOTOR_INVERTED_VALUE = InvertedValue.CounterClockwise_Positive;
+    private static final SensorDirectionValue STEER_ENCODER_DIRECTION = SensorDirectionValue.CounterClockwise_Positive;
+    private static final double STEER_ENCODER_RANGE = 0.5;
```

Comment:
discontinuity point. If you don't like it, get used to it

Reply:
Why? That just makes it sound confusing for no reason.

Reply:
It doesn't. If you look at the javadoc it explains what a discontinuity point is as well. Range is misleading because when it goes out of the range it would stop sensing. Discontinuity point is the proper term for it